### PR TITLE
Add owner information in GetObjectACL and add Context

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
This PR add the object owner information in the `ObjectInfo` returned by `GetObjectACL`.
And add the context to `GetObjectACL` by creating a method `GetObjectACLWithContext`

### CODE REVIEW

File: api-get-object-acl-context.go [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+		return nil, err
+	}
+
+	objInfo, err := c.statObject(ctx, bucketName, objectName, StatObjectOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	objInfo.Owner.DisplayName = res.Owner.DisplayName
+	objInfo.Owner.ID = res.Owner.ID
```
Comment (pierre-emmanuelJ) [inline api-get-object-acl-context.go]: This is the change to add owner information (L72-L73)

**Comment by poornas** [comment]
@pierre-emmanuelJ, it would be better to do a `git mv get-object-acl.go get-object-acl-context.go` to preserve the history better rather than add and remove files. GetObjectACL() function needs to be replaced with GetObjectACLWithContext in the functional tests and examples/s3/getobjectacl.go

**Comment by pierre-emmanuelJ** [comment]
Thank you, I'll do it ASAP

**Comment by pierre-emmanuelJ** [comment]
> Thank you, I'll do it ASAP
Hello, I'll fix the comment of @poornas  today.
Thank you

**Comment by pierre-emmanuelJ** [comment]
Is it okay for you? 😃

**Pull request merged**